src/responder/common/negcache.c

```diff
@@ -665,6 +667,159 @@ int sss_ncache_set_cert(struct sss_nc_ctx *ctx, bool permanent,
     return ret;
 }
 
+static char *domain_lookup_type_str(TALLOC_CTX *mem_ctx,
+                                    struct sss_domain_info *dom,
+                                    const char *lookup_type)
+{
+    return talloc_asprintf(mem_ctx,
+                          "%s/%s/%s",
```

Comment:
The indentation looks a bit odd here, looks like a white-space is missing.

Reply:
fixed